README.md

```diff
@@ -37,6 +37,8 @@ docker run
     -e COORDINATES=-8,13 // optional, this will overwrite COUNTRY env var
     -e TRANSPORTATION='driving+ferry' //optional
     -e TRAVEL_TIME=7200 //optional
+    -e LEVEL_OF_DETAILS=2 // optional
+    -e RPM=200 // optional
```

Comment:
Wouldn't it make sense to add the `-e RPM=200 // optional` line for other requests in README as well?

Reply:
yeap, we'll need to extend readme